### PR TITLE
Working port with openpgp 4.4.5 and add file encryption

### PR DESCRIPTION
**Pull request by oz123**
Hi Alex,

thanks for putting this page online. 
I updated this page to work with openpgp 4.4.5 and added the ability to encrypt a file.

I am not a javascript professional, so I hope my code isn't terrible.

Oz

### CODE REVIEW

**Comment by oz123** [comment]
actually, I broke the code for encrypting the message.

File: index.html [hunk 1]
```diff
@@ -1,138 +1,176 @@
...
-  query('#private-key').className = 'visible';
+	query('#private-key').className = 'visible';
 });
 
 function getKey(textarea, type) {
-  const { value } = textarea;
-  if (!value) {
-    throw new Error(`${type} key missing`);
-  }
-  return openpgp.key.readArmored(value).keys[0];
```
Comment (alexgorbatchev) [inline index.html:99]: Curious why was this removed? `getKey` now doesn't really do much.

**Comment by alexgorbatchev** [comment]
I think the change to `getKey` what broke the message encrypting.

File: index.html (hunk 1 above)
Comment (oz123) [reply index.html:99]: Hi Alex, 
as I warned before... I am a JS noob. I had troubles with the new async API.
So I removed the key reading to it's own block. I removed the Error throwing too just out of frustration I guess.
I now pushed a new working implementation. You can find it in my Fork. 
This implementation can also encrypt a file. But I am still trying to figure out how to rename the file properly.
Bummer ... I think my code is still broken ...

Comment (oz123) [reply index.html:99]: my current code works if I debug it. So I think I am doing something wrong with Promises. But I don't grok anything with async await in JS yet. So I let it be for the while...
